### DISCUSSION TITLE
Roboto font is used in the code?

### DISCUSSION THREAD

**Discussion by rohandebsarkar** [alshedivat/al-folio#441] [Q&A] [answered]
While checking the PageSpeed Insights, I noticed there is a font file loading from fonts.googleapis.com. I tried to find where the font is mentioned in the code, but could not find it. The font seems to the Roboto font.
![Screenshot 2021-10-31 at 14-55-25 PageSpeed Insights](https://user-images.githubusercontent.com/50144004/139576223-87754132-6943-4fc0-879d-dd93cf738f23.png)

**Comment by alshedivat** [comment, marked as answer]
good point. the font might be used in the SVG: https://github.com/alshedivat/al-folio/search?q=roboto

**Reply by rohandebsarkar** [reply]
In that case, I don't think we need to include that in the code, as it is not required by anything on the website.

**Reply by alshedivat** [reply]
sorry, it looks like Roboto is actually used by Material Bootstrap. you can see it if you inspect the page of the demo website:
<img width="502" alt="Screen Shot 2021-11-01 at 9 33 40 AM" src="https://user-images.githubusercontent.com/2126561/139680030-354ddcda-ceb5-45e4-aa72-cc1f16c33e05.png">

**Reply by rohandebsarkar** [reply]
Okay.
